yash-datalake/nested/blobTriggerdFunction.json

```diff
@@ -0,0 +1,275 @@
+            },
+            {
+              "name": "elasticclusterstorage_STORAGE",
+              "value": "[Concat('DefaultEndpointsProtocol=https;AccountName=',parameters('storage_name'),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', parameters('storage_name')),'2018-07-01').keys[0].value)]"
+            },
+            {
+              "name": "esIp",
+              "value": "[parameters('EsIP')]"
+            },
+            {
+              "name": "kibanaFunctionHost",
+              "value": "[parameters('kibanaFunctionHost')]"
+            },
+            {
+              "name": "kibanaFunctionPath",
+              "value": "[parameters('kibanaFunctionPath')]"
+            },
+            {
+              "name": "raweventhubnamespace_RootManageSharedAccessKey_EVENTHUB",
+              "value": "[concat(listkeys(resourceId('Microsoft.EventHub/namespaces/authorizationRules', variables('eventgridEventHubNamespaceName'), 'RootManageSharedAccessKey'), '2017-04-01').primaryConnectionString,';EntityPath=',variables('raweventhub'))]"
+            },
+            {
+              "name": "raweventhub",
+              "value": "[variables('raweventhub')]"
+            }
+          ]
+        },
+        "reserved": false,
+        "scmSiteAlsoStopped": false,
+        "clientAffinityEnabled": false,
+        "clientCertEnabled": false,
+        "hostNamesDisabled": false,
+        "containerSize": 1536,
+        "dailyMemoryTimeQuota": 0
+      },
+      "dependsOn": [
+        "[resourceId('Microsoft.Web/serverfarms', parameters('serverfarms_EastUS2Plan_name'))]",
+        "[concat('Microsoft.EventHub/namespaces/', variables('eventgridEventHubNamespaceName'))]",
+        "[variables('raweventhub')]"
+      ],
+      "resources": [
+        {
+          "name": "[variables('pythonExtension')]",
+          "type": "siteextensions",
+          "apiVersion": "2018-02-01",
+          "properties": {},
+          "dependsOn": [
+            "[resourceId('Microsoft.Web/sites',parameters('sites_demotriggerapp1234_name'))]"
+          ]
+        },
+        {
+          "name": "MSDeploy",
+          "type": "extensions",
+          "location": "[parameters('location')]",
+          "apiVersion": "2018-02-01",
+          "dependsOn": [
+            "[concat('Microsoft.Web/sites/', parameters('sites_demotriggerapp1234_name'))]",
+            "[resourceId('Microsoft.Web/serverfarms', parameters('serverfarms_EastUS2Plan_name'))]",
+            "[resourceId('Microsoft.Web/sites/siteextensions', parameters('sites_demotriggerapp1234_name'), variables('pythonExtension'))]"
+          ],
+          "tags": {
+            "displayName": "pythonextension"
+          },
+          "properties": {
+            "packageUri": "https://github.com/ajos1993/yash-datalake/raw/master/scripts/elasticsearchfunction.zip",

```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts

Reply:
This is the same as in azureDeploy.

>>If we use deployment().properties.templateLink.uri() or other method as shown in Best 
    practices then it is unable to refer our nested templates since right now we are working on 
    forked repo. I think once its in Azure repo it will work. Can you suggest any other change 
    which will work out in the current situation?

Reply:
Updated according to best practices